src/components/Documents/DocumentsMobile.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+
+import { useTheme } from '@mui/material/styles';
+
+import DocumentCard from './DocumentCard';
+
+/**
+ * @typedef {object} Document
+ * @property {string} name - The given name of the document
+ * @property {string} type - The given type of the document
+ * @property {string} description - The given description of the document
+ * @property {string} uploadDate- The upload date of the document
+ * @property {string} endDate - The expiration date of the document
+ * @property {string} fileUrl - The file URL of the document
+ */
+
+/**
+ * @typedef {object} Handlers
+ * @property {Function} onPreview - Function to handle document previewing.
+ * @property {Function} onShare - Function to handle document sharing.
+ * @property {Function} onDelete - Function to handle document deletion.
+ */
+
+/**
+ * DocumentsMobile - Component for displaying documents in a DataGrid
+ *
+ * @memberof Documents
+ * @name DocumentsDesktop
+ * @param {object} Props - The props for DocumentsMobile
+ * @param {Document[]} Props.documents - The list of documents to display
+ * @param {Handlers} Props.handlers - Object containing event handler functions.
+ * @returns {React.JSX.Element} The DocumentsMobile component
+ */
+const DocumentsMobile = ({ documents, handlers }) => {
+  const theme = useTheme();
+
+  // parent box element
+  const tableHeaderStyling = {
+    display: 'flex',
+    justifyContent: 'space-between'
+  };
+
+  return (
+    <Box>
+      <Box
+        sx={{
+          my: '15px',
+          p: '15px',
+          background: theme.palette.primary.main,
+          color: '#fff',
+          borderRadius: '8px',
+          position: 'relative'
+        }}
+      >
+        <Box sx={tableHeaderStyling}>
+          <Typography sx={{ marginLeft: '0' }}>Name</Typography>
+          <Typography>Type</Typography>
```

Comment:
I'm not sure there's enough room for this Type column.

![2024-08-26 (2)](https://github.com/user-attachments/assets/3c7df642-c7e9-4c09-8e4b-8f0478051d04)

It's competing for space with the document name and, as Kim pointed out, it's not aligned.

I think that top level should just be Name and Actions, like in the Contacts table. The other data, like upload and expiration dates, can be listed underneath vertically. It'd also make the actions menu centered in the card, which I think is more appealing.

Here's a very quick mockup just to give an idea:

![2024-08-26 (3)](https://github.com/user-attachments/assets/cae9ec5f-0910-42e6-9b60-b7f75d76eae5)